frontend/chat-plugin/example.html

```diff
@@ -208,10 +208,11 @@
             w[n] = w[n] || {};
             w[n].cid = search.get("channel_id");
             w[n].h = "chatplugin.airy";
+            w[n].tls = false;
```

Comment:
Small fix: For **local use cases only** we need to allow the plugin to turn off tls